README.md

```diff
@@ -301,6 +301,19 @@ const auth = new Auth({
 });
 ```
 
+You may also choose to pass an access token from a Taskcluster-Login-supported authentication service,
+e.g. auth0. Passing an access token will cause the client instance to remotely fetch the associated
+credentials, which will then be filled in as options just as though they had been passed directly to
+the client constructor.
+
+Passing an access token to the client constructor also has the added benefit of automatically fetching
+new credentials when the existing credentials expire, up until the access token's valid duration.
```

Comment:
This should probably have an example.  Let's leave the custom URL and exchange function undocumented for the moment, though